ci/pipelines.release.yml

```diff
@@ -0,0 +1,91 @@
+updateCommitStatus: &UPDATE_COMMIT_STATUS update_commit_status jenkinsReleaseGit --context "$step_name"
+
+pipelines:
+  - name: build_jenkins
+    configuration:
+      runtime:
+        type: image
+        image:
+          auto:
+            language: java
+            versions:
+              - "8.0.0"
+      environmentVariables:
+        readOnly:
+          NEXT_VERSION: 0.0.0
+          NEXT_DEVELOPMENT_VERSION: 0.0.x-SNAPSHOT
+
+    steps:
+      - name: Release
+        type: Bash
+        configuration:
+          inputResources:
+            - name: jenkinsReleaseGit
+          integrations:
+            - name: ojo
+            - name: github_generic
+            - name: jenkins_id_rsa
+        execution:
+          onStart:
+            - *UPDATE_COMMIT_STATUS
+            - restore_cache_files m2_home ${HOME}/.m2
+          onExecute:
+            - cd $res_jenkinsReleaseGit_resourcePath
+
+            # Set env
+            - export CI=true
+            - export M2_HOME=/usr/local/apache-maven-3.6.3
+            - export JFROG_CLI_BUILD_NAME=artifactory-jenkins-plugin
+            - export JFROG_CLI_BUILD_NUMBER=$run_number
+
+            # Configure git
+            - git checkout master
+            - git remote set-url origin https://$int_github_generic_token@github.com/jfrog/jenkins-artifactory-plugin.git
+            - ssh-add ~/.ssh/jenkins_id_rsa
+            - git remote add upstream git@github.com:jenkinsci/artifactory-plugin.git
+
+            # Make sure versions provided
+            - echo "Checking variables"
+            - test -n "$NEXT_VERSION" -a "$NEXT_VERSION" != "0.0.0"
+            - test -n "$NEXT_DEVELOPMENT_VERSION" -a "$NEXT_DEVELOPMENT_VERSION" != "0.0.0"
+
+            # Download JFrog CLI
+            - curl -fL https://getcli.jfrog.io | sh && chmod +x jfrog
+
+            # Configure JFrog CLI
+            - ./jfrog rt c jfrog --url $int_jfrog_rt_url --user $int_jfrog_rt_user --password $int_jfrog_rt_password
+            - ./jfrog rt c ojo --url $int_ojo_rt_url --access-token=$int_ojo_rt_token
+            - ./jfrog rt mvnc
+              --server-id-resolve ojo --repo-resolve-releases jfrog-dependencies --repo-resolve-snapshots jfrog-dependencies
+              --server-id-deploy jfrog --repo-deploy-releases releases --repo-deploy-snapshots snapshots
+
+            # Sync changes with upstream
+            - git pull upstream master
+
+            # Update version
+            - ./jfrog rt mvn versions:set -DnewVersion="${NEXT_VERSION}" -B
```

Comment:
I think it is better to use ```mvn``` without jfrog-cli for this command.
(and also for the other ```versions:set``` command).